app/views/account/login/login2021.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
```

Comment:
Did you intend to delete this when you were done with the PR?

Reply:
Ah, yes. Let's delete. I didn't know the origin or need for this file.